grub/scripts/quick-install-themes.sh

```
@@ -3,7 +3,7 @@
 # Quick Theme Installer for GRUB
 # Downloads and installs popular GRUB themes
 
-THEMES_DIR="/home/whodoe/.config/grub"
+THEMES_DIR="$HOME/.config/grub"
```

Shellcheck (0.10.0)

[warning] THEMES_DIR appears unused. Verify use (or export if used externally).

(SC2034)